packages/figma-utils/src/commands/import-variables.ts

```diff
@@ -49,14 +50,17 @@ export const importCommand = new Command("import-variables")
  */
 export async function importCommandAction(options: ImportCommandOptions) {
   const generators = {
-    CSS: (data: ParsedVariable) => generateAsCSS(data, options.selector),
+    CSS: (data: ParsedVariable) => generateAsCSS(data, { selector: options.selector }),
     SCSS: generateAsSCSS,
+    JSON: generateAsJSON,
   };
 
-  if (!(options.format in generators)) {
-    throw new Error(
-      `Unknown format: ${options.format}. Supported: ${Object.keys(generators).join(", ")}`,
-    );
+  for (const format of options.format) {
+    if (!(format in generators)) {
+      throw new Error(
+        `Unknown format "${format}". Supported: ${Object.keys(generators).join(", ")}`,
+      );
+    }
   }
```

Comment:
I'd suggest that we consistently prefer .forEach over for ... of, as it is a lot more performant. (I know here we only have 3 turns, but consistency 😇)

```suggestion
  options.format.forEach((format) => {
    if (!(format in generators)) {
      throw new Error(
        `Unknown format "${format}". Supported: ${Object.keys(generators).join(", ")}`,
      );
    }
  });
```

Reply:
if you want you can also update Line 74 with the modes checks, seems I missed that one on the last review.